clients/ui/frontend/src/app/pages/modelCatalog/components/ValidatedModelCard.tsx

```diff
@@ -0,0 +1,247 @@
+  const validatedLabels = allLabels.includes('validated') ? ['validated'] : [];
+
+  const metrics = extractValidatedModelMetrics(performanceMetrics, accuracyMetrics);
+
+  return (
+    <Card
+      isFullHeight
+      data-testid="validated-model-catalog-card"
+      key={`${model.name}/${model.source_id}`}
+    >
+      <CardHeader>
+        <CardTitle>
+          <Flex alignItems={{ default: 'alignItemsCenter' }} style={{ paddingLeft: '8px' }}>
+            {model.logo ? (
+              <img
+                src={model.logo}
+                alt="model logo"
+                style={{ height: '56px', width: '56px', marginRight: '8px' }}
+              />
+            ) : (
+              <Skeleton
+                shape="square"
+                width="56px"
+                height="56px"
+                screenreaderText="Brand image loading"
+                style={{ marginRight: '8px' }}
+              />
+            )}
+            <FlexItem align={{ default: 'alignRight' }}>
+              <Badge className="pf-m-purple">Validated</Badge>
+            </FlexItem>
+          </Flex>
+          <div style={{ marginTop: '8px' }}>
+            <Flex alignItems={{ default: 'alignItemsCenter' }} gap={{ default: 'gapSm' }}>
+              <div
+                data-testid="validated-model-status-indicator"
+                className={text.textColorStatusSuccess}
+                style={{
+                  width: '8px',
+                  height: '8px',
+                  backgroundColor: 'var(--pf-t--global--palette--green-400)',
+                  borderRadius: '50%',
+                }}
+              />
+              <Content component={ContentVariants.small} className={text.textColorStatusSuccess}>
+                Success
+              </Content>
+            </Flex>
+            <Link to={catalogModelDetailsFromModel(model.name, source?.id)}>
+              <Button
+                data-testid="model-catalog-detail-link"
+                variant="link"
+                tabIndex={-1}
+                isInline
+                style={{ fontSize: '1.25rem', fontWeight: 'bold' }}
+              >
+                {truncate ? (
+                  <span>{getModelName(model.name)}</span>
+                ) : (
+                  <span>{getModelName(model.name)}</span>
+                )}
+              </Button>
+            </Link>
+          </div>
+        </CardTitle>
+      </CardHeader>
+      <CardBody>
+        <div>
+          <div style={{ paddingLeft: '8px', marginBottom: '16px' }}>
+            <Flex
+              alignItems={{ default: 'alignItemsCenter' }}
+              justifyContent={{ default: 'justifyContentSpaceBetween' }}
+            >
+              <Flex
+                alignItems={{ default: 'alignItemsCenter' }}
+                gap={{ default: 'gapSm' }}
+                wrap="nowrap"
+              >
+                <ChartBarIcon />
```

Comment:
This is the wrong icon, use the `MonitoringIcon`. [Icon docs](https://www.patternfly.org/design-foundations/icons#all-icons)

<img width="862" height="189" alt="Screenshot 2025-10-07 at 5 02 06 PM" src="https://github.com/user-attachments/assets/c6216932-e397-4f0d-b030-b45c29c71962" />